Modules/_operator.c

```diff
@@ -1018,6 +1025,7 @@ static int
 itemgetter_traverse(itemgetterobject *ig, visitproc visit, void *arg)
 {
     Py_VISIT(ig->item);
+    Py_VISIT(Py_TYPE(ig));
```

Comment:
nitpick: I would prefer to visit the type first ;-) same remark for other traverse functions.